app/views/provider/admin/referrer_filters/create.js.erb

```diff
@@ -19,7 +19,7 @@
 
   <%# Hide form if limit reached. %>
   <% if @cinstance.referrer_filters.size >= ReferrerFilter::REFERRER_FILTERS_LIMIT %>
-    $(".enabled_block").fadeOut()
-    $(".disabled_block").fadeIn()
+    $("#referrer-filters-form").hide()
+    $("#referrer-filters-limit").show()
```

Comment:
We could keep `.enabled_block`/ `.disabled_block` here, but I thought more explicit names were better.

Reply:
I also think that hide-show is cleaner than the fadeOut-fadeIn animation, which I think it a bit clunky.